esx_service/vmdk-opsd

```diff
@@ -30,6 +28,14 @@ MAX_QUICK_FAILURES=5
 SIGTERM=15
 SIGKILL=9
 
+# Config
+VMDK_OPSD_PORT=1019 # Override using CONFIG_FILE
+
+# Create the following file if defaults need to be overridden
+# Example:
+# export VMDK_OPSD_PORT=1020
+CONFIG_FILE=/etc/vmdk_opsd/config.sh
```

Comment:
log config file is a standard python logging config, we just reuse it. 

I guess we can add something to json there as long as it is ignored by logging subsystem, or filter it out on read ourselves - but this adds more moving parts for questionable value IMO. I am not against this, just pointing out it's extra work and I am not sure it's worth it